app/javascript/packs/application.js

```diff
@@ -81,25 +81,20 @@ $(function() {
   });
 
   // Toggle the side navigation
-  $('#sidebarToggle, #sidebarToggleTop').on('click', function(e) {
-    $('.sidebar').toggleClass('toggled');
+  $('#sidebarToggle, #sidebarToggleTop').on('click', function() {
+    $('.sidebar, #sidebarToggleTop').toggleClass('toggled');
   });
 
-  // On small screens don't show sidebar on load
-  if ($(window).width() < 768) {
-    $('.sidebar').toggleClass('toggled');
-  }
```

Comment:
das musste noch weg. hab beim erneuten testen festgestellt, dass beim navigieren mobil auf der folgeseite dann die sidebar aufgeklappt war über dem content, was hierdurch geschah.

Reply:
das ist aber wahrscheinlich auch was, das nach dem merge in 206 dann anders ist. da habe ich nämlich allgemein das aufklappen beim seiten wechseln verhindert

Reply:
habe schon auf 206 rebased